frontend/public/components/default-resource.jsx

```diff
@@ -117,10 +56,74 @@ export const DetailsForKind = (kind) =>
 export const DefaultList = (props) => {
   const { kinds } = props;
 
+  const { t } = useTranslation();
+
+  const TableHeader = () => {
+    return [
+      {
+        title: t('public~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('public~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+      },
+      {
+        title: t('public~Created'),
+        sortField: 'metadata.creationTimestamp',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[2] },
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[3] },
+      },
+    ];
+  };
+
+  const TableRowForKind = ({ obj, index, key, style, customData }) => {
+    const kind = referenceFor(obj) || customData.kind;
+    const menuActions = [...Kebab.getExtensionsActionsForKind(kindObj(kind)), ...common];
+
+    return (
+      <TableRow id={obj.metadata.uid} index={index} trKey={key} style={style}>
+        <TableData className={tableColumnClasses[0]}>
+          <ResourceLink
+            kind={customData.kind}
+            name={obj.metadata.name}
+            namespace={obj.metadata.namespace}
+            title={obj.metadata.name}
+          />
+        </TableData>
+        <TableData className={classNames(tableColumnClasses[1], 'co-break-word')}>
+          {obj.metadata.namespace ? (
+            <ResourceLink
+              kind="Namespace"
+              name={obj.metadata.namespace}
+              title={obj.metadata.namespace}
+            />
+          ) : (
+            t('public~None')
+          )}
+        </TableData>
+        <TableData className={tableColumnClasses[2]}>
+          <Timestamp timestamp={obj.metadata.creationTimestamp} />
+        </TableData>
+        <TableData className={tableColumnClasses[3]}>
+          <ResourceKebab actions={menuActions} kind={kind} resource={obj} />
+        </TableData>
+      </TableRow>
+    );
+  };
+
   return (
     <Table
       {...props}
-      aria-label="Default Resource"
+      aria-label={t('public~Default Resource')}
```

Comment:
Not new, but the label should really be the actual kind instead of "Default Resource"